Community-Supported/flights-data-incremental-refresh/README.md

```diff
@@ -0,0 +1,48 @@
+# flights-data-incremental-refresh
+## __Incremental Refresh using the OpenSkyApi__
+
+![Community Supported](https://img.shields.io/badge/Support%20Level-Community%20Supported-53bd92.svg)
+
+This sample is based on the content the Hyper team presented in the Hands on Training session "Hands-on: Leverage the Hyper Update API and Hyper API to Keep Your Data Fresh on Tableau Server" at Tableau Conference 2022 ([slides available here](https://mkt.tableau.com/tc22/sessions/live/430-HOT-D1_Hands-onLeverageTheHyperUpdate.pdf)).
+
+It demonstrates how to implement an incremental refresh based on the Hyper API and the [Hyper Update API](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_how_to_update_data_to_hyper.htm). It showcases this based on fligths data from the [OpenSkyAPI](https://github.com/openskynetwork/opensky-api). The sample should serve as a starting point for anyone looking to incrementally publish and update datasources on Tableau Server/Cloud. 
+
+# Get started
+
+## __Prerequisites__
+To run the script, you will need:
+- Windows, Linux, or Mac
+- Python 3
+- Run `pip install -r requirements.txt`
+- Additionally, you need to install the Python OpenSkyApi. You have to manually install it via `pip install https://github.com/openskynetwork/opensky-api/archive/master.zip#subdirectory=python`
+- Tableau Server Credentials
+
+## Tableau Server Credentials
+To run this sample with your Tableau Server/Cloud, you first need to get the following information and copy it into the respective variables at the end of the script:
+- Tableau Server Url, e.g. 'https://us-west-2a.online.tableau.com'
+- Site name, e.g., use 'default' for your default site
```

Comment:
recommending to use Tableau Online, in combination with the "default" site won't work.
E.g., my "default site" is alpo-dev. Still I have to write `alpo-dev` here and can't just write "default"